browser/css/cool.css

```diff
@@ -742,10 +763,10 @@ body {
 	box-sizing: content-box !important;
 	max-width: 32px;
 	display: inline-block;
-	border: solid 2px;
+	border: solid 1px;
```

Comment:
The colorful border cannot be glue to the image otherwise images will blend too much with the outer border and both the border color and the edges of the avatar will be confusing/indistinguishable. Please add also a white outline of 1px that it's offset by -2px.